server/athenian/api/models/state/__main__.py

```diff
@@ -0,0 +1,20 @@
+import os
+import sys
+
+import jinja2
+
+
+def main():
+    """Initialize the server state DB."""
```

Comment:
Or maybe it's to be run at first run to configure the ini file?

Reply:
It initializes the DB if it does not exist, and migrates to the latest version. So that's a shorthand for

- Create the INI file
- Run `alembic upgrade head`

I added this shorthand mostly to simplify the deployment.

Adding the comments about what I've just written.